purchase_stock_secondary_unit/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Jarsa Sistemas
+# Copyright 2021 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Purchase Stock Secondary Unit",
+    "summary": "Get product quantities in a secondary unit",
+    "version": "16.0.1.1.0",
+    "development_status": "Beta",
+    "category": "stock",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "author": "Jarsa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "purchase_order_secondary_unit",
+        "stock_secondary_unit",
```

Comment:
This addon belongs to another OCA repository. Would you consider creating the appropriate [oca_dependencies.txt](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#185oca_dependenciestxt)?